RELEASE_TEMPLATE.md

```diff
@@ -169,13 +172,162 @@ $ echo $?
+$ docker ps | awk '{print $1}' | grep -v CONTAINER | xargs docker stop
+$ git checkout release-$$blox_version$$
+$ cd <GitRepoBase>/deploy/docker/conf/
+$ sed -i '' 's#bloxoss/cluster-state-service:$$blox_version$$#$$ecr_css_repo_uri$$:$$blox_version$$#g' docker-compose.yml
+$ sed -i '' 's#bloxoss/daemon-scheduler:$$blox_version$$#$$ecr_ds_repo_uri$$:$$blox_version$$#g' docker-compose.yml
+$ sed -i '' 's/<region>/$$aws_region$$/g' docker-compose.yml
+$ docker-compose up -d
+$ docker ps
+CONTAINER ID   IMAGE                                   STATUS
+70d2ca6c5de7   $$ecr_ds_repo_uri$$:$$blox_version$$    Up      <- Should show 'Up' and version '$$blox_version$$' 
+e2214884f981   $$ecr_css_repo_uri$$:$$blox_version$$   Up      <- Should show 'Up' and version '$$blox_version$$'
+088f0d7c20e8   quay.io/coreos/etcd:v3.x.y              Up      <- Should show 'Up'
+$ git checkout docker-compose.yml  # Revert local changes to file
+```
+
+#### Ensure that the cluster-state-service end-to-end tests pass against the $$ecr_css_repo_uri$$:$$blox_version$$ image
+- Perform the `Start the ECR images for e2e and integration testing` steps above
+- Perform the `Ensure that the cluster-state-service end-to-end tests pass` steps above
+
+#### Ensure that the daemon-scheduler end-to-end tests pass against the $$ecr_ds_repo_uri$$:$$blox_version$$ image
+- Perform the `Start the ECR images for e2e and integration testing` steps above
+- Perform the `Ensure that the daemon-scheduler end-to-end tests pass` steps above
+
+#### Ensure that the daemon-scheduler integration tests pass against the $$ecr_ds_repo_uri$$:$$blox_version$$ image
+- Perform the `Start the ECR images for e2e and integration testing` steps above
+- Perform the `Ensure that the daemon-scheduler integration tests pass` steps above
+
+#### Push the release-$$blox_version$$ branch up to the Github dev and master branches
+```
+$ git checkout release-$$blox_version$$
+$ git remote add upstream git@github.com:blox/blox.git
+$ git push -v upstream 'release-$$blox_version$$:dev'
+$ git push -v upstream 'release-$$blox_version$$:master'
+```
+
+#### Publish the v$$blox_version$$ release in GitHub
+```
+Open up a new browser window to [here](https://github.com/blox/blox/releases)
+Click on the Edit button next to 'Release v$$blox_version$$'
+Ensure the following details:
+ Tag version: v$$blox_version$$
+ Target: master
+ Release title: Release v$$blox_version$$
+ Description: Release v$$blox_version$$
+ This is a pre-release: Yes
+Click: Publish release
+```
+
+#### Publish the bloxoss/cluster-state-service:latest,$$blox_version$$,$$github_hash$$ images to Docker Hub
+```
+$ docker pull $$ecr_css_repo_uri$$:$$blox_version$$
+$ docker tag $$ecr_css_repo_uri$$:$$blox_version$$ bloxoss/cluster-state-service:latest
+$ docker tag $$ecr_css_repo_uri$$:$$blox_version$$ bloxoss/cluster-state-service:$$blox_version$$
+$ docker tag $$ecr_css_repo_uri$$:$$blox_version$$ bloxoss/cluster-state-service:$$github_hash$$
```

Comment:
Are we also doing a `docker run bloxoss/cluster-state-service:latest --version` check (and version checks for other tags for dockerhub images) somewhere in this process?